node/pkg/utils/pool/pool.go

```diff
@@ -24,11 +24,11 @@ func (p *Pool) Run(ctx context.Context) {
 	p.IsRunning = true
 
 	for i := 0; i < p.workerCount; i++ {
-		go p.worker(poolCtx)
+		go p.work(poolCtx)
 	}
 }
 
-func (p *Pool) worker(ctx context.Context) {
+func (p *Pool) work(ctx context.Context) {
```

Comment:
I thought function name should be verb rather than noun, since it does some action rather than just a property or a value. how do you think? if it is `worker`, at first sight I thought it should be some sort of struct or value

Reply:
well it seems trivial and minor so I'm not insisting, either is good for me. I'll rollback if you miind